src/installatie/browser.md

```diff
@@ -0,0 +1,23 @@
+# Browserondersteuning
+
+Hieronder staan de ondersteunde desktopbrowsers en versies. Mobiele browsers worden niet ondersteund.
+
+| Browser | OS                | Versie    |
+|---------|-------------------|-----------|
+| Chrome  | Windows/Mac/Linux | Laatste 2 |
+| Firefox | Windows/Mac/Linux | Laatste 2 |
+| Edge    | Windows           | Laatste 2 |
+| Safari  | Mac               | Laatste 2 |
+
+## Gebruik geen adblocker
+
+Sommige adblockers in browsers kunnen een foutmelding in Abacus veroorzaken.
+Zet adblockers daarom uit of voeg Abacus toe aan de lijst met uitzonderingen in de adblocker.
```

Comment:
Buiten scope van deze wijziging, maar we hebben bij password managers best uitgebreide instructies. Moet dat hier dan ook niet?